Makefile.pre.in

```diff
@@ -1945,11 +1956,14 @@ Objects/obmalloc.o: $(srcdir)/Objects/mimalloc/alloc.c \
 		$(srcdir)/Objects/mimalloc/random.c \
 		$(srcdir)/Objects/mimalloc/segment.c \
 		$(srcdir)/Objects/mimalloc/segment-map.c \
+		$(srcdir)/Objects/mimalloc/static.c \
 		$(srcdir)/Objects/mimalloc/stats.c \
+		$(srcdir)/Objects/mimalloc/page-queue.c \
```

Comment:
Please make sure you add lines so the list is still alphabetically sorted. Moreover, `page-queue.c` is already a dependency of the `page.o` subtarget; I think we should remove that target, as `page.c` is `#include`d in the source, not compiled as a separate source file.